Lib/concurrent/futures/process.py

```diff
@@ -108,10 +108,12 @@ def _python_exit():
 
 
 # On Windows, WaitForMultipleObjects is used to wait for processes to finish.
-# It can wait on, at most, 63 objects. There is an overhead of two objects:
+# It can wait on, at most, 64 objects. There is an overhead of two to three
+# objects:
 # - the result queue reader
 # - the thread wakeup reader
-_MAX_WINDOWS_WORKERS = 63 - 2
+# - sometimes _PyOS_SigintEvent
+_MAX_WINDOWS_WORKERS = 64 - 3
```

Comment:
I'd prefer for the previous line to state that on Windows waiting for processes to finish uses `_winapi.WaitForMultipleObjects`. Ideally, `_winapi.MAXIMUM_WAIT_OBJECTS` would be defined, which you'd have to add in Modules/_winapi.c. Then we'd have:

```python
# It can wait on, at most, MAXIMUM_WAIT_OBJECTS - 1 (63) objects. An index of
# the wait list is reserved for the interpreter's SIGINT event. There is an
# additional overhead of two objects:
# - the result queue reader
# - the thread wakeup reader
_MAX_WINDOWS_WORKERS = _winapi.MAXIMUM_WAIT_OBJECTS - 3
```

Reply:
I understand the desire to use _winapi.MAXIMUM_WAIT_OBJECTS but as a casual contributor to cpython I don't want to start altering _winapi.c - it's not practical for me to test such changes.

Reply:
You could still use `MAXIMUM_WAIT_OBJECTS` in the comment, and make it clear that the `SIGINT` overhead is due to the `_winapi` implementation, not overhead added by this module. For example:
```python
# In Windows, waiting for processes uses _winapi.WaitForMultipleObjects.
# It can wait on, at most, MAXIMUM_WAIT_OBJECTS - 1 (64 - 1) objects because
# it reserves an index of the wait list for the interpreter's SIGINT event.
# There is an additional overhead of two objects:
# - the result queue reader
# - the thread wakeup reader
_MAX_WINDOWS_WORKERS = 64 - 3
```